app/assets/javascripts/sufia/save_work/save_work_control.es6

```diff
@@ -43,6 +43,17 @@ export class SaveWorkControl {
   }
 
   /**
+   * Keep the form from being submitted many times.
+   *
+   */
+  preventSubmitIfAlreadyInProgress() {
+    this.form.on('submit', (evt) => {
+      if (this.isValid())
```

Comment:
@mjgiarlo yes.  The code above this checks validity on form submit even though you should never be able to submit if the form is invalid.  I was following coding style...